### ISSUE TITLE
Add links in email

### ISSUE THREAD

**Issue by PVince81** [owncloud/activity#580]
For some reason we don't have links in activity emails.

It should use the permalink format "https://host.tld/owncloud/f/$fileId" for every entry in the HTML output.

@pmaier1

**Comment by PVince81** [comment]
see https://github.com/owncloud/core/issues/28227 for an example email output

**Comment by pmaier1** [comment]
Yes, a link would be nice and I would also like to add information about the instance the email was sent from (e.g. 'The following happened at cloud.example.com).

**Comment by pmaier1** [comment]
I gave some input on this here (https://github.com/owncloud/guests/issues/70) as well.

**Comment by PVince81** [comment]
@pmaier1 move priority down to p3. if we want p2 need to discuss

**Comment by ChrisEdS** [comment]
We have an FR from a customer on this topic. Is there an estimation of how and whether we are pursuing this?
